static/app/components/sidebar/index.tsx

```diff
@@ -628,167 +697,187 @@ function Sidebar() {
       {profiling}
       {replays}
       {discover2}
-    </SidebarAccordion>
+    </SubnavMenu>
   );
 
   return (
-    <SidebarWrapper
-      aria-label={t('Primary Navigation')}
-      collapsed={collapsed}
-      hasNewNav={hasNewNav}
-    >
-      <ExpandedContextProvider>
-        <SidebarSectionGroupPrimary>
-          <DropdownSidebarSection
-            isSuperuser={showSuperuserWarning() && !isExcludedOrg()}
-            hasNewNav={hasNewNav}
-          >
-            <SidebarDropdown
-              orientation={orientation}
-              collapsed={hasNewNav || collapsed}
-            />
-
-            {showSuperuserWarning() && !isExcludedOrg() && (
-              <Hook name="component:superuser-warning" organization={organization} />
-            )}
-          </DropdownSidebarSection>
-
-          <PrimaryItems>
-            {hasOrganization && (
-              <Fragment>
-                <SidebarSection hasNewNav={hasNewNav}>
-                  {issues}
-                  {projects}
-                </SidebarSection>
+    <SidebarWrapper>
```

Comment:
Ah i see
iiuc, we're using the `SubnavContainer` component here as the main panel? (then in the children determines, whether to render a secondary panel OR the accordion?)

Do we need this wrapper anymore then or can we move the styles here to the container/primary wrapper?

Also - wonder if we should rename the `SubnavContainer` to just be `NavContainer` 😅